
Some fixes needed to build with gcc >= 3.4.x.

  - Rene Rebe <rene@exactcode.de>

--- ./arch/sparc64/kernel/sparc64_ksyms.c.orig	2004-09-25 00:36:46.680141528 +0200
+++ ./arch/sparc64/kernel/sparc64_ksyms.c	2004-09-25 00:37:11.662343656 +0200
@@ -63,7 +63,7 @@
 extern void die_if_kernel(char *str, struct pt_regs *regs);
 void _sigpause_common (unsigned int set, struct pt_regs *);
 extern void *__bzero(void *, size_t);
-extern void *__bzero_noasi(void *, size_t);
+extern size_t *__bzero_noasi(void *, size_t);
 extern void *__memscan_zero(void *, size_t);
 extern void *__memscan_generic(void *, int, size_t);
 extern int __memcmp(const void *, const void *, __kernel_size_t);
